prov/verbs/src/verbs_eq.c

```diff
@@ -73,14 +85,19 @@ static struct fi_info *
 fi_ibv_eq_cm_getinfo(struct fi_ibv_fabric *fab, struct rdma_cm_event *event,
 		struct fi_info *pep_info)
 {
-	struct fi_info *info, *fi;
+	struct fi_info *info, *ip;
```

Comment:
Why is the rename needed? The name ip doesn't tell if it's an fi_info structure.